tests/test_validate.py

```diff
@@ -34,16 +38,23 @@ def test_validate_token():
     val = validate.validate({'email': user_email, 'oken': token + 'fl'}, None)
```

Comment:
Unrelated to this PR but a small bug fix that can be added alongside: 'oken' should be 'token'